src/doc/python_scripting/visit_utils.rst

```diff
@@ -0,0 +1,134 @@
+visit_utils
+=============
+
+visit_utils is a pure python module distributed along with Visit_'s
+Python interface. It provides a simple interface to encode movie files
+and methods that wrap more complex VisIt Python command sequences
+to simplify a few common use cases. It also provides a stand alone
+PySide based annotation rendering API.
+
+Here we provide details on the encoding and engine launching modules:
+
+visit_utils.encoding
+----------------------
+
+``visit_utils.encoding`` provides methods that allow you to use 
+movie encoders (e.g ffmpeg) to encode movies from sequences of 
+image files and extract image files from movie files.
+
+Methods:
```

Comment:
Am wondering...this is going to wind up being a lot of detail that might be useful to have while actually at the Python prompt. Any easy way to also get into python doc strings, much like we do for other parts of CLI?

Reply:
This was adapted from existing docs strings, so that info is already there.

Reply:
Not sure if there is a magic way we can connect the two, to have them in one place, but this code is pretty solidified I don't expect it to drift much.

Reply:
that's correct, it's using the sphinx specific python doc style.

Reply:
@cyrush does this sphinx functionality help at all with this...

https://www.sphinx-doc.org/en/master/tutorial/automatic-doc-generation.html